### DISCUSSION TITLE
Is it possible to disable playback time on top of screen?

### DISCUSSION THREAD

**Discussion by simurq** [smplayer-dev/smplayer#352] [Q&A] [answered]
Every time I play a video, the player shows playback time on top of the screen, which is really annoying and sometimes, depending on video frame height, covers the view. I've checked under `Preferences → General` and `Interface` to disable this functionality, to no avail yet...

Using SMPlayer v**21.10.0** (revision **10000**) on Pop!_OS **21.10**, Linux Kernel v**5.15.15**

Thanks!

**Comment by smplayer-dev** [comment, marked as answer]
Menu View -> OSD -> Subtitles only.

**Comment by simurq** [comment]
That's helped indeed, thank you! Yet I think it'd be better if this option is e.g. under `Preferences → Subtitles`. It's not quite obvious to look for it in its present location.

**Comment by Maxmillium** [comment]
Would really really like to see a keyboard operated toggle between "show playback time" and "don't show playback time"

**Reply by smplayer-dev** [reply]
You can press "O" several times.
Also "I" displays the playback time for 2 seconds.

**Comment by Maxmillium** [comment]
Hi Ricardo:
Thank you so very much for that - smplayer without some instruction is an enigma.

I also discovered a 2-second bit that appears in the bottom-left corner of the player when using the "[", "]", and "Backspace" keys!

Happy again...!
